crates/fluvio/Cargo.toml

```diff
@@ -40,6 +40,7 @@ siphasher = "0.3.5"
 cfg-if = "1.0.0"
 derive_builder = "0.11.0"
 async-trait = "0.1.51"
+flate2 = "1.0"
```

Comment:
should be optional and depend on `smartengine`

Reply:
Those are independent. `smartengine` feature enables the engine on the client side (precisely, on the producer), whereas `flate2` uses it to decompress SmartModule binary received from the Fluvio cluster. It can be used regardless of the producer.